script/can-i-deploy.js

```diff
@@ -34,7 +34,7 @@ const checkDeployability = async (retries = 3, timeout = 30) => {
   const { verificationResults, canDeploy } = await pactBrokerClient.canDeploy({
     pactDir: pactsFolder,
     version: commitHash,
-    tag: 'master',
+    tag: 'main',
```

Comment:
It doesn't look like we're using this script anymore, but I updated this to be safe